dask/utils_test.py

```diff
@@ -123,3 +129,16 @@ def import_or_none(name):
         return importlib.import_module(name)
     except (ImportError, AttributeError):
         return None
+
+
+def hlg_layer(hlg: HighLevelGraph, prefix: str) -> Layer:
```

Comment:
I could very well be missing something, but it appears this utility isn't used anywhere (outside of the unit test making sure this behaves as expected). I'd prefer to not include it if it's not used anywhere

Reply:
You're right, it's not used in this PR. I was going to use it here once this is merged: https://github.com/dask/dask/pull/8310#discussion_r745065741. Just felt like a generally useful utility for testing things about HLGs.